conjur-oss/templates/postgres.yaml

```diff
@@ -41,6 +41,9 @@ spec:
   template:
     metadata:
       labels: *AppPostgresLabels
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/ssl-cert.yaml") . | sha256sum }}  
```

Comment:
I believe that we can drop this checksum dependency for `ssl-cert.yaml` for postgres. The `ssl-cert.yaml` is used to create Kubernetes secrets for Conjur <-> External World certificates. The certificates that the postgres pod uses are defined in `secrets.yaml`.